Core/GDCore/Extensions/Metadata/ExpressionMetadata.cpp

```diff
@@ -23,7 +23,8 @@ ExpressionMetadata::ExpressionMetadata(const gd::String& returnType_,
       shown(true),
       smallIconFilename(smallicon_),
       extensionNamespace(extensionNamespace_),
-      isPrivate(false) {
+      isPrivate(false),
+      releventContext("Any") {

```

Comment:
Ditto for the other files
```suggestion
      relevantContext("Any") {
```